CHANGELOG.md

```diff
@@ -56,6 +56,11 @@ SPDX-License-Identifier: LGPL-3.0-or-later
 - Add an About Ikarus page in the documentation ([#291](https://github.com/ikarus-project/ikarus/pull/291))
 - Add new class `Vtk::Writer`, which implements some convenience methods over the existing `dune-vtk` module ([#309](https://github.com/ikarus-project/ikarus/pull/309))
 - Add `VanishingStrain` material (useful for example for plane strain case), also refactor the constructor of `LinearElastic` to take any linear material law ([#317](https://github.com/ikarus-project/ikarus/pull/317))
+- Add new result evaluators and support for full stress evaluation for vanishing strain and stress in `LinearElastic` and `NonlinearElastic` ([#343](https://github.com/ikarus-project/ikarus/pull/343)). Further interface changes:
```

Comment:
With so many rebases, this got accidentally removed from #333 and so is seen as a change here. My bad

Reply:
Resolved in #353